### PR TITLE
Adds a few more golem names

### PR DESCRIPTION
**Pull request by 81Denton**
This PR adds a few more special names for golems, nothing worth adding to the CL.

### CODE REVIEW

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 1]
```diff
@@ -937,7 +937,7 @@
 /datum/species/golem/leather
 	name = "Leather Golem"
 	id = "leather golem"
-	special_names = list("Face", "Man")
+	special_names = list("Face", "Man", "Belt") //Ah dude 4 strength 4 stam leather belt AHHH
```
Comment (Tlaltecuhtli) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: no cool references?

**Comment by ExcessiveUseOfCobblestone** [comment]
how does special names work again? me don't want to read code

**Comment by 81Denton** [comment]
Unless overridden, golememes have a 5% chance to gain a special surname and a 3% chance for a human surname instead of the ones in golem.txt

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 2]
```diff
@@ -805,7 +805,7 @@
 	name = "Plastic Golem"
 	id = "plastic golem"
 	prefix = "Plastic"
-	special_names = null
+	special_names = list("Sheet", "Bag")
```
Comment (tralezab) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: bottle?

**Review by SpaceManiac — APPROVED** [review]

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 3]
```diff
@@ -516,7 +516,7 @@
 	attack_verb = "honk"
 	attack_sound = 'sound/items/airhorn2.ogg'
 	prefix = "Bananium"
-	special_names = null
+	special_names = list("Peel","Cream Pie","Split","Honk")
```
Comment (coiax) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: Bananium golems override the `random_name` proc, so these names aren't used.

Comment (81Denton) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]: Thanks, I missed that one.

Comment (ExcessiveUseOfCobblestone) [reply code/modules/mob/living/carbon/human/species_types/golems.dm]:
```DM
/datum/species/golem/bananium/random_name(gender,unique,lastname)
	var/clown_name = (prob(50)? pick(special_names) : pick(GLOB.clown_names))
	var/golem_name = "[uppertext(clown_name)]"
	return golem_name
```

????

File: code/modules/mob/living/carbon/human/species_types/golems.dm [hunk 4]
```diff
@@ -516,7 +516,7 @@
 	attack_verb = "honk"
 	attack_sound = 'sound/items/airhorn2.ogg'
 	prefix = "Bananium"
-	special_names = list("Peel","Cream Pie","Split","Honk")
+	special_names = null
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/mob/living/carbon/human/species_types/golems.dm]: this is the default value so just delete the line entirely OR do what I put in that last review comment >:)

**Review by ExcessiveUseOfCobblestone — CHANGES_REQUESTED** [review]
bop it

**Review by ExcessiveUseOfCobblestone — APPROVED** [review]

**Pull request merged**